shared/wallets/routes.js

```diff
@@ -114,3 +118,79 @@ const routeTree = () => {
 }
 
 export default routeTree
+
+const walletsSubRoutes = isMobile
+  ? {}
+  : {
+      createNewAccount: {getScreen: () => require('./create-account/container').default},
+      exportSecretKey: {getScreen: () => require('./export-secret-key/container').default},
+      linkExisting: {getScreen: () => require('./link-existing/container').default},
+      reallyRemoveAccount: {getScreen: () => require('./wallet/settings/popups').ReallyRemoveAccountPopup},
+      receive: {getScreen: () => require('./receive-modal/container').default},
+      removeAccount: {getScreen: () => require('./wallet/settings/popups').RemoveAccountPopup},
+      renameAccount: {getScreen: () => require('./wallet/settings/popups').RenameAccountPopup},
+      setDefaultAccount: {getScreen: () => require('./wallet/settings/popups').SetDefaultAccountPopup},
+      setInflation: {getScreen: () => require('./wallet/settings/popups').InflationDestination},
+      settings: {getScreen: () => require('./wallet/settings/container').default},
+      transactionDetails: {getScreen: () => require('./transaction-details/container').default},
+      wallet: {getScreen: () => require('./wallet/container').default},
+    }
+
+class WalletsSubNav extends React.PureComponent<any> {
+  render() {
+    const navigation = this.props.navigation
+    const index = navigation.state.index
+    const activeKey = navigation.state.routes[index].key
+    const descriptor = this.props.descriptors[activeKey]
+    const childNav = descriptor.navigation
+    const WalletsAndDetails = require('./wallets-and-details/container').default
```

Comment:
Kinda weird that this is in the render. On purpose?

Reply:
I wanted to defer making this until it was needed.

Reply:
Why not in the constructor then? I guess this is fine, just kinda weird since we don't do this like this anywhere else

Reply:
then i'd have to store if it its in the constructor